File: Lab_2/src/Pieciokat.java
public class Pieciokat extends Figura
{

    double bok;

    Pieciokat (double bok)
    {
        this.bok = bok;
    }

    @Override
    public double count_Pole()
    {
        pole = (Math.pow(bok, 2) / 4) * Math.sqrt(25 + (10 * Math.sqrt(5)));
        pole = Math.round(pole * 10.0) / 10.0;
        return pole;
    }

    @Override
    public double count_Obwod()
    {
        obwod = bok * 5;
        return obwod;
    }
}

File: Lab_2/С++/Pieciokat.cpp
#include "Pieciokat.h"

    Pieciokat::Pieciokat (double bok) : bok(bok) { }

    double Pieciokat::count_Pole()
    {
        pole = (pow(bok, 2) / 4) * sqrt(25 + (10 * sqrt(5)));
        pole = round(pole * 10.0) / 10.0;
        return pole;
    }

    double Pieciokat::count_Obwod()
    {
        obwod = bok * 5;
        return obwod;
    }